src/cache/cache.module.ts

```
@@ -0,0 +1,31 @@
+@Global()
+@Module({
+  providers: [
+    {
+      provide: CACHE_STORE,
+      useFactory: (configService: ConfigService) => {
+        const config = loadCacheConfig((k) => configService.get(k));
+        if (config.provider === 'redis') {
+          if (!config.redisUrl) {
+            throw new Error(
+              'REDIS_URL is required when CACHE_PROVIDER=redis',
+            );
```

ESLint

[error] Replace `⏎··············'REDIS_URL·is·required·when·CACHE_PROVIDER=redis',⏎············` with `'REDIS_URL·is·required·when·CACHE_PROVIDER=redis'`

(prettier/prettier)